starter-code/css/main.css

```diff
@@ -0,0 +1,73 @@
+*{
+  font-family: Lato, sans-serif;
+}
+body{
+  width: 90%;
+  border-left: solid rgba(173, 173, 173, 1) 5px;
```

Comment:
When adding values to the border property, you should specify in this order: width, style, color.